CHANGELOG.md

```diff
@@ -24,6 +24,18 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian-labs/db-replica/compare/release-1.3.0...master
 
+### Added
+- Allow getting cached value from `SuppliedCache` without executing a supplier
+
+### Changed
+
+- Make `PessimisticPropagationConsistency` class final
+- Make `ThrottledCache` class final
+- Make `NoCacheSuppliedCache` class final
```

Comment:
Breaking compatibility is ok 👌 
We just have to communicate it with a major bump. `db-replica:2.0.0` here we come!